hyahatiph-labs/gitpayd
# refactor noOps to API call. Updatd cli args. Temp bug fix failing tests

Closes #10 

### Description

[Description of your changes go here. Please provide both casual and technical explanations.]

### Payment Request

LN:lnbc1500n1psx5waxpp50pfah3thu8qckmerld4sje6g6arn4re9j9n6gdhcynr0jmqphsysdqvg9jxgg8zn2sscqzpgxqr23ssp5c72vkxacy8vsjcq3sh0dmsaz8nfzg3el0pjvmuz85ej3qxesmapq9qyyssqnkzjxum0d86rrtpk8fr8a9k0mt4w952qk502naq9rpyqpggs768hf082n6v93a5u0gmp75wt0ugv0hz54efagd7x3j24ky6mengcd3gqr3ztgn

This repo is powered by the gitpay daemon and LND. Get paid for closing issues
<br/>
Learn more about LND at https://dev.lightning.community/

### Steps to Test

1. Merge a change to branch your repo is using for workflow
2. Verify build succeeds and API call to NoOps is made

### Screenshots

[Only if applicable]
